tests/unit/account_id_test.py

```
@@ -495,13 +632,266 @@ def test_validate_checksum_with_alias_key_set(client, alias_key):
+    """Test validateChecksum should raise ValueError if evm_address is set"""
+    account_id = AccountId.from_string("0.0.100-hhghj")
+    account_id.evm_address = evm_address
+
+    with pytest.raises(ValueError, match="Cannot calculate checksum with an account ID that has a aliasKey or evmAddres"):
+        account_id.validate_checksum(client)
 
 def test_validate_checksum_for_invalid_checksum(client):
     """Test Invalid Checksum for Id should raise ValueError"""
     account_id = AccountId.from_string("0.0.100-abcde")
     
     with pytest.raises(ValueError, match="Checksum mismatch for 0.0.100"):
```

Ruff (0.14.10)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)